src/transformer/events/core/user_loggedin.php

```diff
@@ -39,19 +40,27 @@ function user_loggedin(array $config, \stdClass $event) {
     $repo = $config['repo'];
     $user = $repo->read_record_by_id('user', $event->userid);
     $lang = $config['source_lang'];
+    $ctx_extensions = utils\extensions\base($config, $event, null);
+
+    if (!is_null($event->relateduserid)) {
```

Comment:
Will this be null if no impersonation? Or will it always be "Cliff logged in as Cliff". Approving either way, just curious.

Reply:
yep, it's null if no impersonation